src/components/organizations/OrganizationFilter.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Filter, TextInput } from "react-admin";
+
+const OrganizationFilter = (props) => (
+  <Filter {...props}>
+    <TextInput label="Search by Name" source="organizationName" alwaysOn />
+    <TextInput label="Search by Email" source="organizationEmail" />
```

Comment:
Since these are filters, I would suggest that you label them as such. Given that the labels aren't strictly titles, I would also suggest that you only make the first word's first letter uppercase.